database/migrations/2023_04_26_104633_create_landing_pages_tables_from_pages_table.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class CreateLandingPagesTablesFromPagesTable extends Migration
+{
+    public function up()
+    {
+        // Create landing_pages tables
+
+        DB::statement('CREATE TABLE landing_pages AS TABLE pages');
+        DB::statement('CREATE SEQUENCE landing_pages_id_seq AS integer');
+
+        // Use existing 'pages' tables for data structure
+
+        DB::statement('CREATE TABLE landing_page_slugs AS TABLE page_slugs');
+        DB::statement('ALTER TABLE landing_page_slugs RENAME COLUMN page_id TO landing_page_id');
+        DB::statement('CREATE SEQUENCE landing_page_slugs_id_seq AS integer');
+        DB::statement("ALTER TABLE landing_page_slugs ALTER COLUMN id SET DEFAULT nextval('landing_page_slugs_id_seq'::regclass)");
+
+        DB::statement('CREATE TABLE landing_page_revisions AS TABLE page_revisions');
+        DB::statement('ALTER TABLE landing_page_revisions RENAME COLUMN page_id TO landing_page_id');
+        DB::statement('CREATE SEQUENCE landing_page_revisions_id_seq AS integer');
+        DB::statement("ALTER TABLE landing_page_revisions ALTER COLUMN id SET DEFAULT nextval('landing_page_revisions_id_seq'::regclass)");
+
+        DB::statement('CREATE TABLE landing_page_categories AS TABLE page_categories');
+        DB::statement('CREATE SEQUENCE landing_page_categories_id_seq AS integer');
+        DB::statement("ALTER TABLE landing_page_categories ALTER COLUMN id SET DEFAULT nextval('landing_page_categories_id_seq'::regclass)");
+
+        DB::statement('CREATE TABLE landing_page_category_slugs AS TABLE page_category_slugs');
+        DB::statement('CREATE SEQUENCE landing_page_category_slugs_id_seq AS integer');
+        DB::statement("ALTER TABLE landing_page_category_slugs ALTER COLUMN id SET DEFAULT nextval('landing_page_category_slugs_id_seq'::regclass)");
+
+        // Create column in relational tables to match landing_page_id instead of page_id
+
+        DB::statement('ALTER TABLE home_artists ADD COLUMN landing_page_id INTEGER');
+        DB::statement('ALTER TABLE featured_hours ADD COLUMN landing_page_id INTEGER');
```

Comment:
Will `landing_page_id` and `page_id` live side-by-side until we've migrated to the LPB?

Reply:
Sounds good.